### PR TITLE
Video page RHD-621 WIP

### PR DESCRIPTION
**Pull request by wesbos**
Still missing a few pieces of data - to come..

### CODE REVIEW

**Comment by Dantheman720** [comment]
Can one of the admins verify this patch?

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
@wesbos @paulrobinson 
I believe this is failing due to code issues not related to the build. If this is not the case let me know and I'll investigate further

**Comment by paulrobinson** [comment]
Here's a snippet from the error:

Excecuting pipeline...
Unable to decrypt vault (GPGME::Error::DecryptFailed)
Generating pages...
[0;31;49mAn error during rendering /mnt/hudson_workspace/workspace/developers.redhat.com-pull-player-executor/mnt/hudson_workspace/workspace/developers.redhat.com-pull-player-executor/synthetic/video/youtube/mIw_DXu5JF4.html occurred.[0m
[0;31;49mWhile processing file /mnt/hudson_workspace/workspace/developers.redhat.com-pull-player-executor/synthetic/video/youtube/mIw_DXu5JF4.html[0m
[0;31;49mAn error occurred: undefined method `each' for "":String[0m
[0;31;49m/mnt/hudson_workspace/workspace/developers.redhat.com-pull-player-executor/_layouts/video_page.html.slim:51:in `block in singleton class'
/mnt/hudson_workspace/workspace/developers.redhat.com-pull-player-executor/_layouts/video_page.html.slim:1:in `instance_eval'
/mnt/hudson_workspace/workspace/developers.redhat.com-pull-player-executor/_layouts/video_page.html.slim:1:in `singleton class'
/mnt/hudson_workspace/workspace/developers.redhat.com-pull-player-executor/_layouts/video_page.html.slim:-1:in `__tilt_19788600'

**Comment by wesbos** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by wesbos** [comment]
@insectengine  - can you review a video page and let me know your thoughts on the layout. Happy to change any design tweaks you have. 

http://it-developers-pr.stage.redhat.com/pr/496/build/1293/video/youtube/2lg8rbz9buU/

**Comment by ollyjshaw** [comment]
retest this please

**Comment by insectengine** [comment]
@wesbos : Hey Wes, just saw this comment. There are some comments on the JIRA : https://issues.jboss.org/browse/RHD-1055

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
@wesbos Can you please rebase with master. Thanks. cc @paulrobinson

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by wesbos** [comment]
retest this please

**Comment by paulrobinson** [comment]
@wesbos looks like you need to rebase.

**Comment by paulrobinson** [comment]
'Get Started link' on http://it-developers-pr.stage.redhat.com/pr/496/build/187/video/youtube/2lg8rbz9buU/ should go to the EAP get-started page.

**Comment by paulrobinson** [comment]
Looks like this needs rebasing.

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
The acceptance tests failed, but seems to be a connection error.

**Comment by paulrobinson** [comment]
@RubyGemTSL any ideas why this failed? http://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/frontend-acceptance-tests/577/console

**Comment by paulrobinson** [comment]
looks like it-developers-pr.stage.redhat.com is unavailable.

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
I'm seeing empty related video carousels on: http://it-developers-pr.stage.redhat.com/pr/496/build/212/video/vimeo/92554627/

Looks like: https://www.dropbox.com/s/gmm4omevdj3frcs/Screenshot%202015-12-10%2011.23.27.png?dl=0

**Comment by wesbos** [comment]
Looks like the Experiments API was failing and breaking JS so I've added in a check for that.

**Comment by paulrobinson** [comment]
Will that be causing us problems in other places on the site?

**Comment by wesbos** [comment]
No - if anything it's saving problems in any of the JS in the overlay, devnation2015, book, jira, carousel, video JS files

File: _eap-quickstarts [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 8d9b0b373f688a1d04ad50fea78e349721f4da62
+Subproject commit 71a4719e5f52f25857b51b3a4f2e781db5659f45
```
Comment (paulrobinson) [inline _eap-quickstarts]: Can you revert this change please.

File: _layouts/base.html.slim [hunk 2]
```diff
@@ -202,6 +202,7 @@
 
     #fb-root
 
+<<<<<<< 65a63b4fb00067d6f2b5ccda43f9019ea4ed9d39
```
Comment (paulrobinson) [inline _layouts/base.html.slim]: Looks like a merge issue

File: _eap-quickstarts (hunk 1 above)
Comment (wesbos) [inline _eap-quickstarts]: How does this happen? With a rake update? Where do I roll it back?

Comment (paulrobinson) [inline _eap-quickstarts]: I got this from Olly:

You probably will fix most things by doing a

'git submodule update'

Of course if you made and committed the changes in the submodule then you'll need to

'git add _submodule; git commit'

Comment (wesbos) [inline _eap-quickstarts]: Yeah no edits to the sub module - will try that. Thanks

**Comment by wesbos** [comment]
@Dantheman720 is fixing something in his aweplug branch. I'll squash all these commits once were good to go

**Comment by paulrobinson** [comment]
@wesbos is this ready to merge?

**Comment by wesbos** [comment]
@paulrobinson Yep - should I squash the commits? I'd rather leave them unsquashed if I can as they are pretty cleanly related to one thing each.

**Comment by ollyjshaw** [comment]
retest this please
